### PR TITLE
Fix of hicexplorer

### PR DESCRIPTION
**Pull request by joachimwolff**

### CODE REVIEW

File: tools/multiqc/multiqc.xml [hunk 1]
```diff
@@ -202,7 +202,10 @@ mkdir multiqc_WDir &&
         @LN_FILES@
     #elif str($repeat.software_cond.software) == "hicexplorer"
         #set $pattern = "Min rest. site distance"
-        @LN_FILES@
+        #for $i, $file in enumerate($repeat.software_cond.input)
+            grep -q "$pattern" $file || die "'$pattern' not found in the file" &&
```
Comment (bebatut) [inline tools/multiqc/multiqc.xml:206]: Can you replace the `grep` and the `ln` line there by
```
#set file_path = os.path.join($software_dir, str($i))
@CHECK_LN_FILE@
```
to reuse the already defined functions and also use a cleaner path joining

**Review by bebatut — COMMENTED** [review]
Thanks!
I am just afraid we are solving here an issue that is not specific to HiCExplorer

File: tools/multiqc/multiqc.xml (hunk 1 above)
Comment (joachimwolff) [reply tools/multiqc/multiqc.xml:206]: `@CHECK_LN_FILE@` is not defined in the file and the execution of MultiQC quits with `cannot find 'identifier'`. I see that you use this identifier several times, probably I am doing something wrong?

**Comment by joachimwolff** [comment]
We can close this imho.

**Pull request closed without merging**